Binary files ./misc/seqan_instrumentation/last_revision_copy/core/include/seqan/seeds2/.seeds_extension.h.swp and ./core/include/seqan/seeds2/.seeds_extension.h.swp differ
diff -u -r -N -x '*.o' -x Thumbs.db -x .DS_Store -x CMakeCache.txt -x misc/seqan_instrumentation/userdata/id.txt -x /Users/konrad/Projects/workshop11/seqan/misc/seqan_instrumentation/userdata/id.txt -x misc/seqan_instrumentation/userdata/rfwo8250x9sgr52s_stats.txt -x /Users/konrad/Projects/workshop11/seqan/misc/seqan_instrumentation/userdata/rfwo8250x9sgr52s_stats.txt -x .svn -x build -x util -x misc -x docs -x misc/seqan_instrumentation/bin -x /Users/konrad/Projects/workshop11/seqan/misc/seqan_instrumentation/bin -x misc/seqan_instrumentation/last_revision_copy -x /Users/konrad/Projects/workshop11/seqan/misc/seqan_instrumentation/last_revision_copy -x misc/seqan_instrumentation/userdata -x /Users/konrad/Projects/workshop11/seqan/misc/seqan_instrumentation/userdata ./misc/seqan_instrumentation/last_revision_copy/core/include/seqan/seeds2/seeds_extension.h ./core/include/seqan/seeds2/seeds_extension.h
--- ./misc/seqan_instrumentation/last_revision_copy/core/include/seqan/seeds2/seeds_extension.h	2011-09-13 09:37:01.000000000 +0200
+++ ./core/include/seqan/seeds2/seeds_extension.h	2011-09-13 12:15:53.000000000 +0200
@@ -97,14 +97,15 @@
 ..summary:Extends a seed.
 ..cat:Seed Handling
 ..signature:extendSeed(seed, query, database, direction, MatchExtend)
-..signature:extendSeed(seed, query, database, direction, scoreDropOff, scoreMatrix, {UngappedXDrop, GappedXDrop})
+..signature:extendSeed(seed, query, database, direction, scoreMatrix, scoreDropOff, {UngappedXDrop, GappedXDrop})
 ..param.seed: The seed to extend.
 ...type:Class.Seed
 ..param.query: The query sequence.
 ...type:Class.String
 ..param.query: The database sequence.
 ...type:Class.String
-..param.direction: Defines the direction in which the seed should be extended. 0 = left, 1 = right, 2 = both
+..param.direction: Defines the direction in which the seed should be extended.
+...type:Enum.ExtensionDirection
 ..param.scoreDropOff: The score drop after which the extension should stop. The extension stops if this value is exceeded.
 ...remarks:Only used for the algorithms @Tag.Seed Extension.UngappedXDrop@ and @Tag.Seed Extension.GappedXDrop@
 ..param.scoreMatrix: The scoring scheme.
